src/overlays/actors/ovl_En_Ge3/z_en_ge3.c

```diff
@@ -30,37 +57,250 @@ const ActorInit En_Ge3_InitVars = {
     (ActorFunc)EnGe3_Update,
     (ActorFunc)EnGe3_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Ge3/func_80A34620.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Ge3/EnGe3_Init.s")
+static ColliderCylinderInit sCylinderInit = {
+    { COLTYPE_UNK10, 0x00, 0x09, 0x39, 0x10, COLSHAPE_CYLINDER },
+    { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000722, 0x00, 0x00 }, 0x00, 0x01, 0x01 },
+    { 20, 50, 0, { 0, 0, 0 } },
+};
```

Comment:
in top of the other comments mentioned here, id merge master and get this updated